digits/model/images/classification/test_views.py

```diff
@@ -70,6 +70,10 @@ class BaseViewsTest(digits.test_views.BaseViewsTest):
 """
 
     @classmethod
+    def setUpClass(cls):
+        super(BaseViewsTest, cls).setUpClass()
```

Comment:
Yes indeed, it is actually a left over from the torch branch where special action was taken in case this was a torch test. I can remove the method completely for the time being. Thanks.